admin_tests/metrics/test_utils.py

```diff
@@ -52,23 +53,22 @@ class TestMetricsGetDaysStatistics(AdminTestCase):
     def setUp(self):
         super(TestMetricsGetDaysStatistics, self).setUp()
         Node.remove()
-        NodeFactory(category='project')  # makes 2 nodes
-        NodeFactory(category='data')  # probably makes 1 data 1 project?
-        print get_projects()
+        NodeFactory(category='project')  # makes Node, plus parent
```

Comment:
I removed it where it touched on the functionality of the PR (because I could get away with it), but otherwise tried to avoid scope creep.  @Ghalko concurs that `ProjectFactory` would be a better choice in future admin test work.